whittle/models/gpt/blocks/mlp.py

```diff
@@ -22,23 +22,16 @@ def __init__(self, config: Config) -> None:
         self.sub_network_intermediate_size = self.intermediate_size
 
     def set_sub_network(
-        self,
-        sub_network_n_embd: int,
-        sub_network_intermediate_size: int,
-        sample_random_indices: bool = False,
+        self, sub_network_n_embd: int, sub_network_intermediate_size: int
```

Comment:
somehow the formatting seems to have changed? Could you re-run ruff?

Reply:
Ruff doesn't reformat this somehow :(

Reply:
I think the formatting changed, just because there are lesser arguments right now